src/routes/hooks/hooks-notifications.spec.ts

```diff
@@ -2017,7 +2024,7 @@ describe.skip('Post Hook Events for Notifications (Unit)', () => {
       }
     });
 
-    pushNotificationsApi.enqueueNotification
+    notificationsRepository.enqueueNotification
```

Comment:
As we are mocking the repository, it's's not possible to test this as the logic is inside the method.